src/sass/_selectable_main_table.scss

```
@@ -7,6 +7,13 @@
   }
 }
 
+// fix for safari https://warthogs.atlassian.net/browse/WD-7486
+#selectable-main-table {
+  span.p-checkbox__label {
```

```suggestion
  .multiselect-checkbox span.p-checkbox__label {
```
This let's the tests pass, can you ensure it also fixes the issue?

The idea is to limit the change to the heading row of the table.

---

Ah thanks for taking a look. I'll check if it fixes the issue.